src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator';
+import 'bootstrap/dist/css/bootstrap.css';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <div className="container my-4"><Calculator /></div>;
```

Comment:
- [ ] Kindly use class components for the main component as shown on the requirements.

Reply:
This change has been done well